src/pages/take-action.js

```diff
@@ -8,36 +8,113 @@ import ElectrifyEverything from "../images/electrify-everything.jpg"
+              <li>
+                Replace your heating system with an{" "}
+                <a
+                  href="https://en.wikipedia.org/wiki/Heat_pump"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  electric heat pump
+                </a>
+                , your gas stove with an{" "}
+                <a
+                  href="https://en.wikipedia.org/wiki/Induction_cooking"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  induction stove
+                </a>
+                , and your water heater with an electric one
+              </li>
               <li>If you drive, your next car must be electric</li>
               <li>Produce your own power with solar panels</li>
             </ol>
             <p>
-              40% of US climate pollution comes from decisions made around the kitchen table. We can’t solve climate change if you don’t electrify your home – and nobody can do it but you.
+              40% of US climate pollution comes from decisions made around the
+              kitchen table. We can’t solve climate change if you don’t
+              electrify your home – and nobody can do it but you.
+            </p>
+
+            <p>
+              Check out Rewiring America's free guide{" "}
+              <a
+                href="https://www.rewiringamerica.org/electrify-home-guide"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Electrify Everything in Your Home
+              </a>{" "}
+              for steps on how to get started.
             </p>
 
-            <p>Check out Rewiring America's free guide <a href="https://www.rewiringamerica.org/electrify-home-guide" target="_blank" rel="noreferrer">Electrify Everything in Your Home</a> for steps on how to get started.</p>
+            <div>
+              <h3 className="mb-2">
+                <strong>
+                  Congress just passed a new bill making all of these cheaper!
+                </strong>
+              </h3>
+              With the passing of the Inflation Reduction Act, there are now{" "}
+              <a
+                href="https://www.fastcompany.com/90780252/how-to-decarbonize-your-house-inflation-reduction-act-incentives?partner=rss&utm_campaign=rss+fastcompany&utm_content=rss&utm_medium=feed&utm_source=rss"
+                target="_blank"
+                rel="noreferrer"
+              >
+                new national rebates for electrification of household
+                appliances.
+              </a>
+              <div>
```

Comment:
we should use a `<ul>` element here instead of a `<div>`. This will fix the left margin: 

![Screen Shot 2022-08-23 at 9 53 09 PM](https://user-images.githubusercontent.com/919583/186309920-9470d700-8d8d-46a8-8781-f7d56940ba89.png)